docs/DesignDocs/Attestation_API_Proposal.md

```diff
@@ -591,63 +594,119 @@ plugin libraries, composed of:
   - It also implements OCALLs for the enclave-side library to access
   host-side services for SGX quote verification.
 
-The OE SDK V0.10 release will also have enclave-side and host-side plugins
-for verification of SGX ECDSA reports generated by the legacy API function
-`oe_get_report()` and raw SGX ECDSA quotes generated by the Intel SGX SDK.
-These two evidence formats are identified by their UUIDs:
-- `OE_FORMAT_UUID_SGX_ECDSA_P256_REPORT`: for SGX ECDSA report generated
-by `oe_get_report()`.
-- `OE_FORMAT_UUID_SGX_ECDSA_P256_QUOTE`: for raw SGX ECDSA quote generated
-by the Intel SGX SDK.
+Currently there is no plan to implement verifier plugins for verification of
+EPID evidence. Verification of SGX EPID quotes is not supported by
+the Intel SGX SDK. In all existing SGX EPID solutions, EPID quotes are sent to
+Intel Attestation Service (IAS) backend for verification.
+
+The OE SDK V0.1x release will also have enclave-side and host-side plugins
+for verification of selected types of OE reports for SGX generated by the legacy
+API function `oe_get_report()` and selected types of SGX quotes generated by
+the Intel SGX SDK DCAP and quote-ex libraries. Note: the plan is for
+the attestation API function `oe_verify_evidence()` to only support a minimum
+set of legacy OE reports and SGX quotes as needed. There is no intention for
+`oe_verify_evidence()` to support verification of all possible types of legacy
+OE reports and SGX quotes.
+
+- `OE_FORMAT_UUID_LEGACY_REPORT_REMOTE`
+  - For OE reports generated by the legacy API function `oe_get_report()`
+  with the `OE_REPORT_FLAGS_REMOTE_ATTESTATION` flag.
+  - Note: no plan to provide verifier plugin for verification of OE reports
+  generated by the legacy API function `oe_get_report()` without the
+  `OE_REPORT_FLAGS_REMOTE_ATTESTATION` flag (for local attestation).
+  These reports can be verified using legacy API function `oe_verify_report()`.
+- `OE_FORMAT_UUID_RAW_SGX_QUOTE_ECDSA`
+  - For SGX quotes in ECDSA_P256 format generated by the Intel SGX SDK DCAP
+  library, and the Intel SGX quote-ex library with algorithm ID
+  `SGX_QL_ALG_ECDSA_P256`.
+    - Note: the DCAP library only generates ECDSA-p256 quotes
+  - Note: no plan to provide verifier plugin for verification of SGX quotes
+  generated by the quote-ex library with other algorithm IDs (e.g. EPID quotes).
```

Comment:
```suggestion
  generated by the quote-ex library with other algorithm IDs (e.g., EPID quotes).
```

Reply:
In response to
> I tend to agree with Simon on this one.

As commented previously:

> I think we can explore various options when we are ready to deprecate oe_get_report(). As Simon mentioned, I'd like this PR to focus on enabling the attestation service scenarios.

Let me know if this approach will not work.